packages/ui/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import type { JSX as Jsx } from 'react/jsx-runtime';
+
+export declare global {
+    namespace JSX {
+        type ElementClass = Jsx.ElementClass;
+        type Element = Jsx.Element;
+        type IntrinsicElements = Jsx.IntrinsicElements;
+    }
+}
```

Comment:
Why is this needed? 🤔

Reply:
Global JSX namespace was dropped in React 19, so external components not yet upgraded to React 19 throw a typescript error, this fixes it